### DISCUSSION TITLE
Why is there no "latest" tag in lighninglabs/lnd image

### DISCUSSION THREAD

**Discussion by martini1992** [lightningnetwork/lnd#7027] [Q&A] [answered]
Would be useful to have a "latest" tag on the lightninglabs/lnd image which would track the latest git tagged release and be the (semi-)stable version of the "daily-testing-only" tag.
I've opened this as a discussion rather than an issue because you are probably already aware of this and have not implemented it for good reason.

**Comment by guggero** [comment, marked as answer]
Because updating `lnd` should be a conscious decision (operators should read the release notes to see what database migrations are applied and what config changes might be required), I don't think a tag that always points to the latest released version is a good idea. Otherwise things might change unexpectedly from one day to the other.
You could argue that the tag should only be used on testnet or regtest. But if there's a tag, people will go ahead and use it on mainnet as well.

Another argument against a static name tag that's updated regularly is that it does not work with repositories that disallow pushing the same tag twice with different hashes. That's why in addition to the `daily-testing-only` tag we also added a `daily-testing-<date>` tag that we use for our testing environments.
